src/Files.Shared/Extensions/GroupOptionExtensions.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) 2023 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+using Files.Shared.Enums;
+
+namespace Files.Shared.Extensions
+{
+	public static class GroupOptionExtensions
+	{
+		public static bool IsGroupByDate(this GroupOption groupOption)
+			=> groupOption is GroupOption.DateModified or GroupOption.DateCreated or GroupOption.DateDeleted;
```

Comment:
Same

Reply:
This is a one-line return, so leave it as is.